.github/workflows/python-package.yml

```
@@ -88,27 +88,19 @@ jobs:
           - windows-latest
           - macos-latest
         python-version:
-          - '3.8'
           - '3.9'
           - '3.10'
           - '3.11'
           - '3.12'
-          - '3.13-dev'
+          - '3.13'
         constraints: ['']
         post_install: ['']
-        exclude:
-          # C extension builds failing.
-          # Remove exclusions after Python 3.13 release.
-          - os: windows-latest
-            python-version: '3.13-dev'
-          - os: macos-latest
-            python-version: '3.13-dev'
         include:
```

I wonder if this will be needed for `3.14-dev`...
```suggestion
        exclude:
          # C extension builds failing.
          # Remove exclusions after Python 3.14 release.
          - os: windows-latest
            python-version: '3.14-dev'
          - os: macos-latest
            python-version: '3.14-dev'
        include:
```